docs/release-notes/Percona-Everest-0.10.0-(2024-05-03).md

```diff
@@ -0,0 +1,103 @@
+# What's new in Percona Everest 0.10.0
+
+To begin your journey with Percona Everest, check out the [Quickstart Guide for Percona Everest](../quickstart-guide/quick-install.md).
+
+Percona Everest is an open source cloud native database platform that helps developers deploy code faster, scale deployments rapidly, and reduce database administration overhead. Plus, you can regain control over your data, database configuration, and DBaaS costs.
+
+Version 0.10.0 introduces the following changes:
+
+
+## Release highlights
+
+### Percona Everest upgrade
+
+!!! warning "Important"
+    You need to download CLI version >=0.10.0 for the upgrade command to work.    
+
+We're thrilled to announce that you can now upgrade your Percona Everest instance using our Command Line Interface (CLI). The CLI upgrade process is simple and straightforward, enabling you to quickly upgrade your database to the latest version.
+
+You can only upgrade one minor version at a time. For instance, you can upgrade from version 0.9.0 to version 0.10.0, but not directly from version 0.8.0 to version 0.10.0.
```

Comment:
We should mention the upgrade works only from version 0.9.0 and higher

Reply:
Will add it.